conmon-rs/server/src/child_reaper.rs

```diff
@@ -341,6 +355,29 @@ impl ReapableChild {
         Ok((exit_tx, exit_rx))
     }
 
+    async fn spawn_cleanup_process(raw_cmd: &mut Vec<String>) -> Result<()> {
```

Comment:
I'd say we can remove the `Result` because we always succeed here:

```diff
diff --git a/conmon-rs/server/src/child_reaper.rs b/conmon-rs/server/src/child_reaper.rs
index 7324251c..35874ab1 100644
--- a/conmon-rs/server/src/child_reaper.rs
+++ b/conmon-rs/server/src/child_reaper.rs
@@ -328,12 +328,8 @@ impl ReapableChild {
                     error!(pid, "Could not write exit paths: {:#}", e);
                 }
 
-                if !cleanup_cmd_raw.is_empty()
-                    && Self::spawn_cleanup_process(&mut cleanup_cmd_raw)
-                        .await
-                        .is_ok()
-                {
-                    debug!("Successful execution of cleanup command process");
+                if !cleanup_cmd_raw.is_empty() {
+                    Self::spawn_cleanup_process(&mut cleanup_cmd_raw).await
                 }
 
                 debug!("Sending exit struct to channel: {:?}", exit_channel_data);
@@ -355,7 +351,7 @@ impl ReapableChild {
         Ok((exit_tx, exit_rx))
     }
 
-    async fn spawn_cleanup_process(raw_cmd: &mut Vec<String>) -> Result<()> {
+    async fn spawn_cleanup_process(raw_cmd: &mut Vec<String>) {
         let mut cleanup_cmd = Command::new(raw_cmd.remove(0));
 
         raw_cmd.iter().for_each(|arg| {
@@ -375,7 +371,6 @@ impl ReapableChild {
                 ),
             }
         });
-        Ok(())
     }
 
     fn wait_for_exit_code(token: &CancellationToken, pid: u32) -> i32 {
```